modules/libffi/3.4.7/overlay/BUILD.bazel

```diff
@@ -0,0 +1,14 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "configure_make_variant")
+
+filegroup(
+    name = "all_srcs",
+    srcs = glob(["**"]),
+)
+
+configure_make_variant(
+    name = "libffi",
```

Comment:
Is it hard to add a BUILD file for this library instead of using rules_foreign_cc? But I'm fine with starting with this version.

Reply:
A BUILD file would need to have a lot of branches for different platforms, which I would like to avoid for now in the first version.